app/src/main/java/com/karhoo/samples/uisdk/dropin/config/KarhooConfig.kt

```diff
@@ -43,4 +43,8 @@ class KarhooConfig(val context: Context, private val authMethod: AuthenticationM
     override fun analyticsProvider(): AnalyticProvider? {
         return null
     }
+
+    override suspend fun requireSDKAuthentication(callback: () -> Unit) {
+
```

Comment:
Shouldn't all these have implementations in order for the client app to work properly when testing over longer periods of time?